frontend/src/components/mainPage/scene/subcomponents/SceneObjectsPanel.vue

```diff
@@ -9,8 +9,8 @@
             @click:close="deleteSceneObject(sceneObject.id)"
         >
             {{ sceneObjectLabel(sceneObject) }}
-            <span class="subObjectStyle" v-if="sceneObject.subobjects && sceneObject.subobjects.length > 0">
-                &nbsp;[{{ sceneObject.subobjects.length }}]
+            <span class="subObjectStyle">
```

Comment:
Chyba nie do końca się zrozumieliśmy :D. v-if powinien być, tylko zamiast warunku powinien wywoływać funkcję hasSubobject. W środku spana powinno być tak jak było wcześniej (oprócz tej spacji). Miałem na myśli, żęby tak to wyglądało:
`<span class="subObjectStyle" v-if="hasSubObject(sceneObject)">`
`[{{ sceneObject.subobjects.length }}]`
`</span>`
A funkcja hasSubObject powinna wtedy wyglądać tak:
`hasSubObject() {`
`  return (sceneObject) => {`
`    return !isEmpty(sceneObject.subobjects);`
`  }`
`}`

Popraw to i mogę zaakceptować zadanie :D.

I taka ogólna rada jak jakaś funkcja jest nazwana `hasSomething` lub `isSomething` itp. i ta funkcja zwraca coś innego niż booleana, to na gdzieś na świecie umiera mały kotek. A nie chcemy mieć małych kotków na sumieniu xDD